dbos/_sys_db.py

```diff
@@ -1906,7 +1842,6 @@ def init_workflow(
             wf_status, workflow_deadline_epoch_ms = self._insert_workflow_status(
```

Comment:
Since it's a single query now. Do we still need to call engine.begin()? Can we just get a connection and use autocommit?